packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -5,18 +5,29 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+
 class SimpleExpansionPanelListTestWidget extends StatefulWidget {
   const SimpleExpansionPanelListTestWidget({
     Key key,
     this.firstPanelKey,
     this.secondPanelKey,
     this.canTapOnHeader = false,
+    this.expandedHeaderPadding
   }) : super(key: key);
 
   final Key firstPanelKey;
   final Key secondPanelKey;
   final bool canTapOnHeader;
 
+  /// If null the below default is applied
```

Comment:
It's nice to be a little more specific in the event that `defaultExpandedHeaderPadding` is no longer below this line in the source code for whatever reason.
```suggestion
  /// If null, the default [ExpansionPanelList]'s expanded header padding value is applied via [defaultExpandedHeaderPadding].
```